src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -501,3 +504,28 @@ export const showPtsdAssaultConclusion = form =>
   form['view:upload781aChoice'] === 'answerQuestions' &&
   (_.get('view:selectablePtsdTypes.view:mstPtsdType', form, false) ||
     _.get('view:selectablePtsdTypes.view:assaultPtsdType', form, false));
+
+export const ancillaryFormUploadUi = (label, itemDescription) =>
+  fileUploadUI(label, {
+    itemDescription,
+    hideLabelText: !label,
+    fileUploadUrl: `${environment.API_URL}/v0/upload_supporting_evidence`,
+    fileTypes: ['pdf', 'jpg', 'jpeg', 'png'],
```

Comment:
Same here too...Although i guess you can do a spread and overwrite these if needed.